parcel_test.go

```diff
@@ -31,57 +33,120 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	// настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	if err != nil {
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```

Reply:
ну только тут у меня наверно должен быть nil, ведь я ожидаю что открою базу без ошибок, а вы мне наверно скинули просто пример.

Reply:
Я перепутал тут комментарии, тут должен быть такой:

Конструкция `require.NoError` не нуждается в оборачивание IF, это самостоятельная конструкция, как и остальные тестовые функции

Нужно исправить во всех тестахer